config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-1.yaml

```diff
@@ -156,6 +168,10 @@ periodics:
   decorate: true
   decoration_config:
     gcs_credentials_secret: "" # Use workload identity for uploading artifacts
+  rerun_auth_config:
```

Comment:
Are there a few missing here? e.g. `periodic-cluster-api-e2e-release-1-1` or are they excluded for a reason? There's a few more throughout that don't have the rerun config added

Reply:
Thx, nice catch.

I think I found 2 that were missing it.

Should be fine now (47x interval, 47x rerun_auth_config)